frontend/src/components/forms/ResumeSourceTracePanel.module.css

```
@@ -114,3 +152,49 @@
+   フォントサイズは zoom 倍率を em で受け取り、見出し/リスト等は em 相対で追従させる。 */
+.markdownBody {
+  flex: 1;
+  min-height: 0;
+  overflow: auto;
+  border: 1px solid var(--border);
+  border-radius: 6px;
+  padding: 0.8rem 1rem;
+  background: var(--bg-input);
+  color: var(--text-primary);
+  line-height: 1.7;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)